app/uk/gov/hmrc/customsservicestatus/services/CustomsServiceStatusService.scala

```diff
@@ -46,11 +46,12 @@ class CustomsServiceStatusService @Inject()(customsServiceStatusRepository: Cust
     }
   }
 
-  def listAll: Future[models.Services] = {
-    val servicesFromConfig: Services = ConfigSource.default.loadOrThrow[Services]
-    customsServiceStatusRepository.findAll() map { services =>
-      val serviceNames = servicesFromConfig.services.map(_.name)
-      models.Services(services.filter(s => serviceNames.contains(s.name)) map (CustomsServiceStatusWithDesc(_)))
+  def listAll: Future[models.Services] =
+    customsServiceStatusRepository.findAll() map { servicesFromRepo =>
```

Comment:
Similarly, can we just go back to calling  `servicesFromRepo` "services"? Understand why you renamed it, but I don't think we need to specify that these came from the DB; happy to keep the name "servicesFromConfig" below to make the distinction.

Reply:
done